src/main/java/mate/controller/car/CreateCarController.java

```diff
@@ -0,0 +1,45 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/add")
+public class CreateCarController extends HttpServlet {
+    private static final String ALL_CARS_PATH = "/cars";
```

Comment:
This variable is duplicated several times, make public and reuse it. 
IMHO: in this case making all params and paths constant is not obligatory since they are pretty readable and don't generate code duplication